frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -272,12 +272,28 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
               }
             />
             {isColorLayer && layer != null ? (
-              <ColorSetting
-                label="Color"
-                value={Utils.rgbToHex(layer.color)}
-                onChange={_.partial(this.props.onChangeLayer, layerName, "color")}
-                className="ant-btn"
-              />
+              <Row className="margin-bottom" align="top">
```

Comment:
I'd change the order from:
"Invert Color"  [switch] "Color" [color picker]
to
"Color" [color picker]  "Inverted"  [switch]

That way, the inverted-label doesn't need to contain the word "color".